src/main/resources/fr/greencodeinitiative/java/ecocode_profile.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ecoCode",
+  "ruleKeys": [
```

Comment:
this list is static. could we use a dynamic list got from code ?

Reply:
We could create a JSON file at compile time.
But for now, we can start by testing if the profile contains all the rules implemented by this plugin.
We can improve this in a next PR?